src/api/app/services/project_status_controller_service/project_status_failure_command_finder.rb

```diff
@@ -0,0 +1,7 @@
+module ProjectStatusControllerService
+  class ProjectStatusFailureCommandFinder < AttribValuesFinder
```

Comment:
This is the only class which differs from the third parameter passed to the `new` method in `call`. To be consistent with the other classes, it should be `ProjectStatusPackageFailCommentFinder`.